.circleci/config.yml

```diff
@@ -329,3 +345,4 @@ workflows:
                - master
    jobs:
       - dependencies-pr
+      - build.docs
```

Comment:
I only realized this now, as the docs job seemed to have been missing from this PR.

We still need docs builds on all PRs to be sure they build fine, we just don't want to publish them.